packages/shared/src/WebView.js

```diff
@@ -21,12 +21,16 @@ export default class WebView extends React.Component<Props> {
     />
   );
 
-  render = () => (
-    <NativeWebView
-      bounces={false}
-      startInLoadingState={false} // it feels faster because we don't have to wait until the page is fully loaded
-      renderError={this.renderError}
-      {...this.props}
-    />
-  );
+  render = () => {
+    console.warn(this.props.source);
```

Comment:
I added this on purpose. Not sure if it's a good idea though...

Reply:
I am not a very big fan of this yellow noisy warnings. How about a console.log instead, less obtrusive?

Reply:
I will remove it completely.